src/GAP.jl

```diff
@@ -24,8 +24,18 @@ include(deps_jl)
 """
     FFE
 
-Wraps a pointer to a GAP FFE immediate object.
+Wrap a pointer to a GAP FFE immediate object.
```

Comment:
Perhaps we should also explain what "FFE" stands for (even if this is only internal documentation)
```suggestion
Wrap a pointer to a GAP FFE (= "finite field element") immediate object.
```